lib/kubernetes-deploy/statsd.rb

```diff
@@ -4,23 +4,40 @@
 
 module KubernetesDeploy
   class StatsD
+    extend ::StatsD
+
+    PREFIX = "KubernetesDeploy"
+
     def self.duration(start_time)
       (Time.now.utc - start_time).round(1)
     end
 
     def self.build
-      ::StatsD.default_sample_rate = 1.0
-      ::StatsD.prefix = "KubernetesDeploy"
+      self.default_sample_rate = 1.0
```

Comment:
Isn't this the default behavior?

Reply:
Actually this suffers from the same problem as `StatsD.prefix` in that it's statically referenced in the underlying library so won't have an effect here (`options[:sample_rate] is used to override if ever we need). I'll remove this line as well.